server/src/main/java/io/spine/server/BoundedContext.java

```diff
@@ -539,4 +458,117 @@ public String toString() {
         return spec.name()
                    .getValue();
     }
+
+    /**
+     * Provides access to features of the context used internally by the framework.
+     *
+     * @throws SecurityException
+     *         if called from outside the framework
+     */
+    @Internal
+    public InternalAccess internalAccess() {
+        Security.allowOnlyFrameworkServer();
```

Comment:
We must allow read-only access to the users. I'd even consider making it a non-internal API. It is often used when trying to bind together repositories, e.g. for routing purposes.

Reply:
Users can always arrange dependencies between the repositories before they are added to `BoundedContextBuilder`. Allowing this as a general practice encourages Big Balls of Muds.